iaso/models/microplanning.py

```diff
@@ -10,6 +10,38 @@
 from iaso.utils.models.soft_deletable import SoftDeletableModel
 
 
+class PlanningSamplingResult(models.Model):
+    """Stores the results of sampling runs."""
+
+    planning = models.ForeignKey("Planning", on_delete=models.CASCADE, related_name="sampling_results")
+    task = models.ForeignKey("Task", on_delete=models.SET_NULL, null=True, blank=True, related_name="sampling_results")
+    pipeline_id = models.CharField(max_length=36, null=True, blank=True, help_text="OpenHexa pipeline UUID")
+    pipeline_version = models.CharField(max_length=100, null=True, blank=True, help_text="Pipeline version identifier")
+    pipeline_name = models.CharField(max_length=100, null=True, blank=True, help_text="Pipeline name")
```

Comment:
It's better to avoid blank and null on `CharField`: https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.Field.null


```suggestion
    pipeline_id = models.CharField(max_length=36, blank=True, help_text="OpenHexa pipeline UUID")
    pipeline_version = models.CharField(max_length=100, blank=True, help_text="Pipeline version identifier")
    pipeline_name = models.CharField(max_length=100, blank=True, help_text="Pipeline name")
```